wasmedge_plugin_sdk/src/sdk/plugin.rs

```diff
@@ -189,7 +189,7 @@ impl Placeholder for OptionString {
     fn create_placeholder() -> Box<Self> {
         let length: u32 = 128;
         let buf = vec![0i8; length as usize].into_boxed_slice();
```

Comment:
It might be better to use `vec![0u8; length as usize]`, although in reality, there’s no difference.

Reply:
I could try your suggestion later.